kotlin/set-max-extent/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+    "category": "Maps and scenes",
```

```suggestion
    "category": "Maps and Scenes",
```
This is causing a crash while running it on the sample-viewer